### ISSUE TITLE
when you first log into the admin panel in the user name field contains e-mail

### ISSUE THREAD

**Issue by Akiyamka** [anchorcms/anchor-cms#1049]
### Summary

A brief but detailed explanation of the issue or bug you are reporting.

### Expected Behaviour

- A list of steps you expected to happen in order

### Actual Behaviour

- A list of steps you actually encountered

### Context details (if applicable)

- Anchor version:
- Server setup:
- URL

**Comment by TheBrenny** [comment]
Was this done with Chrome? Becase chrome detects that you're trying to log in on a domain that it has saved credentials for...

**Comment by daviddarnes** [comment]
Good point, could be that as I've had that before. I guess this would be a browser quirk rather than a bug?

**Comment by TheBrenny** [comment]
True. It might be worth checking the field names in the installer and the login page - chrome detects what to auto-insert based off a value on an element. It could be the `name`, but it'd be worth to Google it.. Might double check when I get home.
